Backend/controller/NewWorldScraper/NewWorldScraper.js

```diff
@@ -19,11 +18,8 @@ async function fetchAllCategories(){
 
   let allCategoriesConfig = {
     method: 'get',
-    maxBodyLength: Infinity,
-    url: url,
-    headers: { 
-      'Cookie': '__cf_bm=mSagMqawJXFKGXyGlx3F.PjlC8JQXz.leHE1tODhuGM-1687867900-0-AYsGTN7Y0dvsC1RuU3cBYA79kKdgWyoz6+f6sOvtXQwq1LfNM+6vZpvUI1a+ZaiyoL25uJhHBp3b5HSu0SkUw6T8ZmnfVopd/yu+ep3F4W7L'
```

Comment:
Cookies are unnecessary, and we may as well have a limit on the maximum body length